### DISCUSSION TITLE
什么时候更新啊

### DISCUSSION THREAD

**Discussion by f4844844235869** [antvis/XFlow#399] [Q&A]
基于安antd5,和x6 2.0

**Comment by xueyibokong** [comment]
同问，现在仓库的2.0分支还是在开发中吗？有没有发布计划呢？

**Reply by hrzzz** [reply]
同问，2.0开发文档啥时候更新，或者今后的计划是咋样的呢？

**Reply by DanielLeefu** [reply]
2.0.4 版本文档已经更新 https://x6.antv.antgroup.com/xflow/guide/introduction
